packages/firebase_auth/firebase_auth_web/lib/firebase_auth_web.dart

```diff
@@ -402,7 +402,14 @@ class FirebaseAuthWeb extends FirebaseAuthPlatform {
       // origin as a single string
       delegate.useAuthEmulator('http://$host:$port');
     } catch (e) {
-      throw getFirebaseAuthException(e);
+      final String code = (e as dynamic).code;
+      // this catches Firebase Error from web that occurs after hot reloading & hot restarting
+      // ignore: avoid_print
+      print(
```

Comment:
If we hide the exception, it feels counter-productive to add a print saying it was hidden

Reply:
Yeah, I was thinking that 🤔

Reply:
+1 on not having the print